### PR TITLE
Fix failing links in content

### PR DESCRIPTION
**Pull request by mazzystr**
Fix failing links in content.  mkdocs build is now clean.

```
# make run
make: Warning: File 'Makefile' has modification time 0.59 s in the future

Makefile: Stop site
podman rm -f userguide 2> /dev/null; echo

Makefile: Run site
podman run -d --name userguide --net=host -v ./:/srv:ro --mount type=tmpfs,destination=/srv/site localhost/kubevirt-userguide:latest /bin/bash -c "mkdocs build -f /srv/mkdocs.yml && mkdocs serve -f /srv/mkdocs.yml -a 0.0.0.0:8000"
66c82bbb1529e19f49dd80eac6603f7ceefcdee955e7859b39e9f9f819fd6105

[root@fedora user-guide]# podman logs -f userguide
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /srv/site
INFO    -  Documentation built in 1.22 seconds
INFO    -  Building documentation...
WARNING -  Config value: 'dev_addr'. Warning: The use of the IP address '0.0.0.0' suggests a production environment or the use of a proxy to connect to the MkDocs server. However, the MkDocs' server is intended for local development purposes only. Please use a third party production-ready server instead.
INFO    -  Cleaning site directory
INFO    -  Documentation built in 1.15 seconds
[I 210218 03:02:30 server:335] Serving on http://0.0.0.0:8000
INFO    -  Serving on http://0.0.0.0:8000
[I 210218 03:02:30 handlers:62] Start watching changes
INFO    -  Start watching changes
[I 210218 03:02:30 handlers:64] Start detecting changes
INFO    -  Start detecting changes
[I 210218 03:02:45 handlers:135] Browser Connected: http://0.0.0.0:8003/virtual_machines/service_objects/
INFO    -  Browser Connected: http://0.0.0.0:8003/virtual_machines/service_objects/
```

Fixed the following...
* Ooof, mkdocs site_url config didn't get changed to `/user-guide` that caused 100s internal links to fail
* windows_virtio_driver page was misspelled, really?
* handful of  link fixes that we normally do
* handful of content fixes related to broken links

### CODE REVIEW

File: docs/virtual_machines/disks_and_volumes.md [hunk 1]
```diff
@@ -944,7 +944,7 @@ fewer IOThreads than CPU, each IOThread will be pinned to a set of CPUs.
 #### IOThreads with QEMU Emulator thread and Dedicated (pinned) CPUs
 
 To further improve the vCPUs latency, KubeVirt can allocate an
-additional dedicated physical CPU[1](creation/dedicated-cpu.md), exclusively for the emulator thread, to which it will
+additional dedicated physical CPU[1](virtual_machines/virtual_hardware/#cpu), exclusively for the emulator thread, to which it will
```
Comment (phoracek) [inline docs/virtual_machines/disks_and_volumes.md:947]: The link gives me 404

https://deploy-preview-389--kubevirt-user-guide.netlify.app/virtual_machines/disks_and_volumes/virtual_machines/virtual_hardware/#cpu

File: docs/virtual_machines/templates.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ KubeVirt on OpenShift).
 
 By deploying KubeVirt on top of OpenShift the user can benefit from the
 [OpenShift
-Template](https://docs.openshift.org/latest/dev_guide/templates.html)
+Template](https://docs.openshift.com/container-platform/4.6/virt/vm_templates/virt-creating-vm-template.html)
```
Comment (phoracek) [inline docs/virtual_machines/templates.md:9]: The rest of the docs refer to OKD. Can we link OKD here too?

File: docs/virtual_machines/templates.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ KubeVirt on OpenShift).
...
 functionality.
 
 ## Virtual machine templates
 
 ### What is a virtual machine template?
 
 The KubeVirt projects provides a set of
-[templates](https://docs.okd.io/latest/dev_guide/templates.html) to
+[templates](https://docs.okd.io/latest/openshift_images/using-templates.html) to
```
Comment (phoracek) [inline docs/virtual_machines/templates.md:17]: Do we want to have this on latest and risk that it gets out of date again or do we pin it to a specific version?

File: docs/virtual_machines/virtual_machine_instances.md [hunk 4]
```diff
@@ -67,10 +67,10 @@ require a provisioner in your cluster.
...
 
 -   More information about persistent and ephemeral volumes:
-    [Disks and Volumes](creation/disks-and-volumes.md)
+    [Disks and Volumes](disks_and_volumes.md)
 
 -   How to access a VirtualMachineInstance via `console` or `vnc`:
-    [Console Access](usage/graphical-and-console-access.md)
+    [Console Access](graphical_and_console_access.md)
 
 -   How to customize VirtualMachineInstances with `cloud-init`:
```
Comment (phoracek) [inline docs/virtual_machines/virtual_machine_instances.md:75]: This link is dead

**Review by phoracek — CHANGES_REQUESTED** [review]

File: docs/virtual_machines/disks_and_volumes.md (hunk 1 above)
Comment (mazzystr) [reply docs/virtual_machines/disks_and_volumes.md:947]: Investigating that

File: docs/virtual_machines/templates.md (hunk 2 above)
Comment (mazzystr) [reply docs/virtual_machines/templates.md:9]: Sure can

File: docs/virtual_machines/virtual_machine_instances.md (hunk 4 above)
Comment (mazzystr) [reply docs/virtual_machines/virtual_machine_instances.md:75]: Investigating

File: docs/virtual_machines/disks_and_volumes.md (hunk 1 above)
Comment (mazzystr) [reply docs/virtual_machines/disks_and_volumes.md:947]: OK, I see what's happenin' here ... We have half of an internal link and half an external link.  The two halves are creating the failure.

We have 19 cases of this in the user-guide.  I'm going to update them all.

Stand by.

File: docs/virtual_machines/virtual_machine_instances.md (hunk 4 above)
Comment (mazzystr) [reply docs/virtual_machines/virtual_machine_instances.md:75]: OK, I see what's happenin' here ... We have half of an internal link and half an external link.  The two halves are creating the failure.

We have 19 cases of this in the user-guide.  I'm going to update them all.

Stand by.

Comment (mazzystr) [reply docs/virtual_machines/virtual_machine_instances.md:75]: Link is fixed per local test and Netlify

File: docs/virtual_machines/disks_and_volumes.md (hunk 1 above)
Comment (mazzystr) [reply docs/virtual_machines/disks_and_volumes.md]: Link is fixed per local test and Netlify

File: docs/virtual_machines/templates.md (hunk 3 above)
Comment (mazzystr) [reply docs/virtual_machines/templates.md:17]: Templates are only good in Red Hat OpenShift. I searched the OKD latest documentation and it seems the Red Hat OKD links in this page refer to general template use not KubeVirt virtual machine common templates. 

I recommend we change the Red Hat OKD links to Red Hat OpenShift links.

Comment (phoracek) [reply docs/virtual_machines/templates.md:17]: Understood. Thanks for checking

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

**Pull request merged**